CHANGELOG.next.asciidoc

```diff
@@ -61,6 +61,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Mark system process metricsets as running if metrics are partially available {pull}40565[40565]
 - Added back `elasticsearch.node.stats.jvm.mem.pools.*` to the `node_stats` metricset {pull}40571[40571]
 - Add support for snapshot in vSphere virtualmachine metricset {pull}40683[40683]
+- Update fields to use mapstr in vSphere virtualmachine metricset  {pull}40707[40707]
```

Comment:
I don't think this is breaking change, since the old PR is not released for users yet.

Reply:
yeah.

Reply:
My bad. This hsould not be here.
Also, I noticed our other PR changelogs are not in properly placed as well.
Some are in filebeat. Some under breaking change. 
We need to recheck this @kush-elastic @niraj-elastic